packages/playground/src/weblets/tf_nextcloud.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <weblet-layout
+    ref="layout"
+    @mount="layoutMount"
+    :cpu="solution?.cpu"
+    :memory="solution?.memory"
+    :disk="disks.reduce((total, disk) => total + disk.size, rootFilesystemSize)"
+    :ipv4="ipv4"
+    :certified="certified"
+    :dedicated="dedicated"
+    title-image="images/icons/nextcloud.png"
+  >
+    <template #title>Deploy Nextcloud All-in-One Instance </template>
+
+    <d-tabs :tabs="[{ title: 'Config', value: 'config' }]" ref="tabs">
+      <template #config>
+        <input-validator
```

Comment:
```suggestion
<input-validator
```

As this weblet only 1 tab we don't need `d-tabs` component anymore